n2y/plugins/linktopage.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from n2y.blocks import Block
+
+logger = logging.getLogger(__name__)
+
+class LinkToPage(Block):
+    """
+    Replace page link with the content of the linked-to page.
+
+    """
+    def __init__(self, client, notion_data, page, get_children=False):
+
+        super().__init__(client, page, notion_data, get_children)
+
+    def to_pandoc(self):
+        assert self.linked_page_id is not None
+
+        try:
+            self.client.get_page(self.linked_page_id) 
+            # Although out of the scope of the project at hand, get_page_or_database() 
```

Comment:
Good point about links to databases, I hadn't considered that! For now I would just log a warning if someone links to a database and then ignore it.